doc/articles/supported-features.md

```diff
@@ -46,12 +46,12 @@
 - RelativeSource.Self
 
 ### Design Fidelity
-- Text Inlines Binding
```

Comment:
This change might be incorrect. @jeromelaban could you confirm?

Reply:
I would keep inlines here, as it matches the XAML `Inlines` property

Reply:
Sure thing -- I'll revert this change and add it to the dictionary instead 👍

Reply:
✅ Done.